tests/foreman/cli/test_remoteexecution.py

```diff
@@ -488,7 +477,7 @@ def test_positive_run_scheduled_job_template_by_ip(self, fixture_vmsetup, defaul
 
     @pytest.mark.tier3
     @pytest.mark.upgrade
-    def test_positive_run_receptor_installer(self, default_sat):
+    def test_positive_run_receptor_installer(self, target_sat, subscribe_satellite):
```

Comment:
Some significant changes to the test here, @lhellebr @pondrejk  (module assignee and function assignee) please review this backport, should these changes be in 6.10.z?